concourse/pipelines/sap-workload-test.yaml

```diff
@@ -0,0 +1,149 @@
+jobs:
+- name: sap-workload-test
+  plan:
+  - task: generate-unique-id
+    config:
+      platform: linux
+      image_resource:
+        type: registry-image
+        source:
+          repository: busybox
+      run:
+        path: sh
+        args:
+        - -exc
+        - |
+          # TODO - make more unique? guid?
```

Comment:
look at our other examples - we often use date based IDs rather than random ones, which helps also with discovery